content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -226,6 +226,100 @@ rules:
   - sign
 ```
 
+## Normal User
+
+Few steps are required in order to get normal to be able to authenticate and invoke API. First, this user must have certificate issued by the Kubernetes Cluster, and then present that Certificate into the API call as the Certificate Header, or through the kubectl.
```

Comment:
I'm not sure what "to get normal" means in this context.

Reply:
I think it is a mistypo, it should be 'to get normal user'

Reply:
@jimangel, refer to this [commit](https://github.com/kubernetes/website/pull/20443/commits/b2d21616d8c527c7ad3c55d904f96b427a2af654)